WordPress/src/main/java/org/wordpress/android/ui/quickstart/QuickStartType.kt

```diff
@@ -50,4 +50,17 @@ sealed class QuickStartType(val tasks: List<QuickStartTask>) {
         return quickStartStore.getDoneCount(siteLocalId) >= tasks
                 .filter { it.taskType != QuickStartTaskType.UNKNOWN }.size
     }
+
+    companion object {
+        private const val NEW_SITE = "new-site"
+        private const val EXISTING_SITE = "existing-site"
+
+        fun fromString(label: String): QuickStartType {
```

Comment:
Nitpick:  I think we can rename the function to `fromLabel`, WDYT? 🤔

Reply:
Renamed in a97be94